### DISCUSSION TITLE
Microphone button sync with openai.com

### DISCUSSION THREAD

**Discussion by agnostikas** [lencx/ChatGPT#1011] [Ideas]
Apologies if this is already addressed, however it didn't appear automatically, hence this suggestion
There is a popular extension in chrome / edge store to activate voice commands, which appear immediately in chat.openai.com
It would be helpful, since it's ultimately a wrapper to the host in question, that this button would also appear in app.
Thank you for consideration

**Comment by gmy77** [comment]
Indeed, I would really benefit from a voice functionality even in the ChatGPT application. I use it a lot on the web and to be honest, I would use it even more on this fantastic application.

Thank you very much for this hard work.
